pkgs/applications/display-managers/sddm/unwrapped.nix

```
@@ -14,6 +14,7 @@
   systemd,
   xkeyboardconfig,
   nixosTests,
+  docutils ? null,
```

```suggestion
  docutils,
```
that will never be null